V3

Zibbp opened the pull request.

Zibbp committed:
Initial commit

Zibbp committed:
update admin info page

Zibbp committed:
feat(queue): restart tasks support

Zibbp committed:
use new twitch route

Zibbp committed:
update category route

Zibbp committed:
blocked video support

Zibbp committed:
support categories with live streams

Zibbp committed:
remove workflows page

Zibbp committed:
add ability to stop and delete queue items

Zibbp committed:
all-in-one settings page

Zibbp committed:
new docker ci

Zibbp committed:
test

Zibbp committed:
test

Zibbp committed:
test

Zibbp committed:
test

Zibbp committed:
test

Zibbp committed:
test

Zibbp committed:
test

Zibbp committed:
test

Zibbp committed:
test

Zibbp committed:
test

Zibbp committed:
test

Zibbp committed:
test

Zibbp committed:
test

Zibbp committed:
test

Zibbp committed:
fix

Zibbp committed:
fix

Zibbp committed:
include channel name in queue table

Zibbp committed:
fix overlay z-index

Zibbp committed:
allow blocking video ids when deleting a video

Zibbp committed:
feat: regenerate static thumbnail button

russelg commented on src/components/Video/Core/VideoCard.tsx:
```diff
@@ -1,85 +1,51 @@
-  playback?: PlaybackData[];
-  showChannel?: boolean;
-}) => {
+const VideoCard = ({ video, progress = 5, showMenu = true, showChannel = true }: Props) => {
```
doesn't look like actual progress is being used in video cards anymore. all my VODs appear as 5% watched.

Zibbp replied:
This was on my list of todos. Pushed a few commits to both repos that re-implement progress on the video cards.

Zibbp committed:
feat(VideoCard): fetch playback data inside card

Not the greatest solution, it means there's a request per video card on the page. The frontend needs a complete re-work in the future, so it's fine for now

Zibbp committed:
calculate progress

Zibbp committed:
remove temp push logic for beta
